tas/templates/tas/profile_edit.html

```diff
@@ -32,10 +32,18 @@ <h2>Edit Profile <small>| {{ user.username }} </small></h2>
         {% else %}
           <div class="checkbox">
             <label>
-              <input type="checkbox" id="id_request_pi_eligibility" name="request_pi_eligibility">
+              {% if is_student %}
+              <input type="checkbox" id="id_request_pi_eligibility" name="request_pi_eligibility" disabled onchange="pie_request();">
+              {% else %}
+              <input type="checkbox" id="id_request_pi_eligibility" name="request_pi_eligibility" onchange="pie_request();">
+              {% endif %}
               Request PI Eligibility
             </label>
           </div>
+          <div id="directorylinkdiv" style="display:none">
+            <label for="directorylink">Please provide the department directory link for review:</label>
```

Comment:
So this is like, the page listing all the faculty members? Maybe be a bit more specific what this means:

"Please provide a link to your institution's department directory or any other official institution page where information confirming your current appointment can be found."